message-switch-unix.opam

```
@@ -19,7 +19,6 @@ depends: [
   "base-threads"
   "message-switch-core"
   "ppx_deriving_rpc"
-  "xapi-stdext-unix"
```

this should be 
```suggestion
  "xapi-stdext-threads"
```
see https://github.com/xapi-project/xen-api/pull/5875

---

I can rebase this PR after we merge yours.

---

Updated